### ISSUE TITLE
Edit Screen Configuration

### ISSUE THREAD

**Issue by coolcat-creations** [joomla-projects/custom-fields#148]
I succeeded to override the article submission through a Menuitem, but however it takes the default one if i edit the article itself :-/. Since i don´t have a idea how to achieve following view for the whole category and also for editing it i would like to ask for extending the "edit screen configuration" options. 
### Example: No Language, Publishing, WYSIWYG, Metadata

![bildschirmfoto 2016-06-20 um 23 20 25](https://cloud.githubusercontent.com/assets/828371/16210675/996ab44a-373d-11e6-9af8-d0c2d1a22e75.png)
### Possible Solution: Extending this Tab

![bildschirmfoto 2016-06-20 um 23 18 00](https://cloud.githubusercontent.com/assets/828371/16210725/c3e80290-373d-11e6-8208-961e99b84a8d.png)
- Hide WYSIWYG on Frontend / Backend/Both
- Show Custom-Fields in First Tab
- Hide Publishing Params on Frontend / Backend/Both
- Hide Language Params on Frontend / Backend/Both
- Hide Metadata Params on Frontend / Backend/Both

**Comment by laoneo** [comment]
I guess this issue needs to be opened against the CMS repo to get feedback there (except the wish for custom fields tabs). If it get accepted, we can see what we do in custom fields with the new configuration options. Ok for you?

**Comment by coolcat-creations** [comment]
Ok, thanks a lot !!! So i have to open it yes?

**Comment by laoneo** [comment]
Yes. Please reference then this issue in your issue on the CMS repo.

**Comment by coolcat-creations** [comment]
Done... hope it´s well enough described...

**Comment by laoneo** [comment]
Well done. If it get's implemented, then we can start thinking about a way to implement it in custom fields.
